core/services/commander/main.py

```diff
@@ -66,6 +69,24 @@ async def shutdown(response: Response, shutdown_type: ShutdownType, i_know_what_
         logger.exception(error)
         return {"message": f"{error}"}
 
+@app.post("/settings/reset", status_code=status.HTTP_200_OK)
+@version(1, 0)
+async def shutdown(response: Response, i_know_what_i_am_doing: bool = False) -> Any:
+    if not i_know_what_i_am_doing:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Developer, you don't know what you are doing, command aborted.",
+        )
+
+    try:
+        user_config_dir = Path(appdirs.user_config_dir())
+        os.rmdir(user_config_dir)
+        return HTMLResponse(status_code=200)
+    except Exception as error:
+        response.status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
+        logger.exception(error)
+        return {"message": f"{error}"}
```

Comment:
I would add a generic "Could not reset user configuration" before the specific error, so it becomes more clear what's happening.